azure-pipelines.yml

```diff
@@ -193,6 +198,89 @@ stages:
           condition: always()
 
 - ${{ if eq(variables._RunAsPublic, True) }}:
+
+  - stage: Test_CLI_Package_In_Helix_Android
```

Comment:
These stages are basically the same but I couldn't quite get project globbing to work so instead of having a SLN file just to do two runs at once I decided to send android from linux hosted and iOS from mac hosted; both are pretty fast.